packages/graphql-transformers-e2e-tests/src/run.ts

```diff
@@ -0,0 +1,3 @@
+import { setupUserPool } from './setupUserPool';
+setupUserPool('us-west-2_9ZddNgNah', '27t3je7vkn9oli2unkq7lf2g4t')
```

Comment:
Is this safe to have these values in the repo?

Reply:
Nope deleted the UP. Good catch.